LayoutTests/imported/w3c/web-platform-tests/service-workers/cache-storage/cache-match.https.any.js

```diff
@@ -56,8 +56,11 @@ prepopulated_cache_test(simple_entries, function(cache, entries) {
 prepopulated_cache_test(simple_entries, function(cache, entries) {
     return cache.match(new Request(entries.a.request.url, {method: 'HEAD'}))
       .then(function(result) {
-          assert_equals(result, undefined,
-                        'Cache.match should not match HEAD Request.');
+          assert_unreached('Should not reach here, exception expected')
```

Comment:
This change does not seem right, according to https://w3c.github.io/ServiceWorker/#cache-matchall, this should return null when method check fails. 

Note that the tests in LayoutTests/imported/w3c/web-platform-tests/ directory are imported from a public test set (https://web-platform-tests.org), which is usually compliant to standard, so you might want to double check the spec when your change breaks it.